Directory.Packages.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <ItemGroup Label="Libraries">
-    <PackageVersion Include="AWSSDK.SimpleNotificationService" Version="3.7.0" />
-    <PackageVersion Include="AWSSDK.SQS" Version="3.7.0" />
+    <PackageVersion Include="AWSSDK.SimpleNotificationService" Version="3.7.300" />
+    <PackageVersion Include="AWSSDK.SQS" Version="3.7.300" />
```

Comment:
These are the minimum versions that support .NET 8 and AoT, so I think they're worth raising the baseline too as part of getting the batch publishing support.